packages/api-client/src/notification/NotificationAPI/NotificationAPI.ts

```diff
@@ -77,7 +76,7 @@ export class NotificationAPI {
       url: NotificationAPI.URL.NOTIFICATION,
     };
 
-    const response = await this.client.sendJSON<NotificationList>(config, false, abortController);
+    const response = await this.client.sendJSON<NotificationList>(config, false);
```

Comment:
I really hoped we would finally adopt Zod.js, since we touch the endpoints...

Reply:
usage of this endpoint will be deprecated after all clients are migrated to the new async notification system hence I don't think it's worth it here.

Reply:
https://github.com/wireapp/wire-web-packages/pull/7066/commits/713b4ead34ac328da21f9eb18817bff7c616b938